### PR TITLE
Migrate to  asto version 0.2

### PR DESCRIPTION
**Pull request by Sammers21**
@yegor256 , review the pull request, please.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -90,7 +90,13 @@ SOFTWARE.
...
       <groupId>com.yegor256</groupId>
       <artifactId>asto</artifactId>
-      <version>0.1.0</version>
+      <version>0.2.0</version>
+    </dependency>
+    <dependency>
+      <groupId>io.reactivex.rxjava3</groupId>
+      <artifactId>rxjava</artifactId>
+      <version>3.0.0-RC6</version>
+      <scope>compile</scope>
```
Comment (yegor256) [inline pom.xml]: @Sammers21 `compile` is the default one, no need to specify it

File: src/main/java/com/yegor256/goproxy/Goproxy.java [hunk 2]
```diff
@@ -81,98 +86,126 @@ public Goproxy(final Storage stg) {
...
-                new org.cactoos.text.Joined(
-                    "\n",
-                    new Joined<String>(
-                        new ListOf<>(
-                            new String(Files.readAllBytes(list)).split("\n")
-                        ),
-                        new ListOf<>(String.format("v%s", version))
+    public Completable update(final String repo, final String version) {
+        return Completable.defer(
+            () -> {
```
Comment (yegor256) [inline src/main/java/com/yegor256/goproxy/Goproxy.java]: @Sammers21 this code looks very unreadable. Let's use private methods instead. Now it's a mess.

File: src/main/java/com/yegor256/goproxy/Goproxy.java [hunk 3]
```diff
@@ -48,18 +47,24 @@
...
+ *
+ * <pre> goproxy.update("example.com/foo/bar", "0.0.1").subscribe();</pre>
  *
  * That's it.
  *
  * @author Yegor Bugayenko (yegor256@gmail.com)
  * @version $Id$
  * @since 0.1
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle LineLengthCheck (500 lines)
```
Comment (yegor256) [inline src/main/java/com/yegor256/goproxy/Goproxy.java]: @Sammers21 don't do this. This limitation is here for a reason.

**Comment by yegor256** [comment]
@Sammers21 see my comments above

File: src/main/java/com/yegor256/goproxy/Goproxy.java (hunk 2 above)
Comment (Sammers21) [reply src/main/java/com/yegor256/goproxy/Goproxy.java]: Can you check if it is any better now?

**Comment by Sammers21** [comment]
@yegor256, can you look at the pull request one more time?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**